lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1081,6 +1113,8 @@ export class UserAgentApplication {
      * @param accessToken
      */
     protected clearCacheForScope(accessToken: string) {
+        this.logger.verbose("Clearing access token from cache");
+        this.logger.verbosePii(`Access token: ${accessToken}`);
```

Comment:
Can you make this message for specific, e.g. "Clearing cache for access token: ${accessToken}"

Reply:
And actually, may not be a good idea to log the access token itself, I think it would be better to put a  message below that logs `token.key` when it gets removed.

Reply:
Done.